components/menu/README.md

```diff
@@ -305,3 +265,8 @@ Nested menus can be defined by placing a `d2l-menu` inside a `d2l-menu-item`.  F
   <d2l-menu-item text="The Night Sky"></d2l-menu-item>
 </d2l-menu>
 ```
+
+## ACCESSIBILITY
+
+- The `label` property for `d2l-menu` is only required for the root menu
+	- For nested menus, the label is automatically applied based on its parent menu-item
```

Comment:
There might also be some interesting talking points around the keyboard behaviour

Reply:
Yeah, we definitely followed the Aria guidelines for menus and menu-items, including keyboard navigation within menus and drilling in/out of nested menus.

Reply:
I've now added something for the menubar pattern and also the keyboard menu. I've also referred to `nested menus` as `submenus` because that's what the W3C docn called them, but I can switch it to be `nested menus`, if that's preferred.

Reply:
Awesome. For the link text, I would probably use "menu and menubar" as per the pattern title, or just "menu". Mostly since it's really the "menu" part of the pattern that we're following... we don't have an implementation of the menubar.